python-client/giskard/datasets/__init__.py

```diff
@@ -1,17 +1,23 @@
 from typing import Dict, Optional, List
 import pandas as pd
+from rich.console import Console
 from giskard.datasets.base import Dataset
 from giskard.core.validation import configured_validate_arguments
 
 
 @configured_validate_arguments
-def dataset(df: pd.DataFrame,
-            name: Optional[str] = None,
-            target: Optional[str] = None,
-            cat_columns: Optional[List[str]] = None,
-            infer_column_types: Optional[bool] = False,
-            column_types: Optional[Dict[str, str]] = None):
-    return Dataset(df,
+def wrap_dataset(dataset: pd.DataFrame,
+                 name: Optional[str] = None,
+                 target: Optional[str] = None,
+                 cat_columns: Optional[List[str]] = None,
+                 infer_column_types: Optional[bool] = False,
+                 column_types: Optional[Dict[str, str]] = None):
+    console = Console()
```

Comment:
I see that you're using a 3rd party library `rich` here. This library isn't directly required by Giskard, but installed as a transitive dependency of `bandit` which is a static analyser (so not installed as part of Giskard's prod dependencies either). There are 2 options:
1. We decide to keep `rich` in which case we should declare it as a depencency of GIskard
2. We stick to standard python print or logging. It's not as convenient to color things, but doesn't require an extra dependency

For now, I'd stick to option 2 to reduce the number of dependencies

Reply:
you're right. Done. Kept it simple with a `print` for now.